requirements.txt

```diff
@@ -2,3 +2,6 @@ Django>=3.2,<4.0
 djangorestframework>=3.13.1,<4.0
 djangorestframework-simplejwt
 django-filter>=21.1,<22.0
+drf-spectacular>=0.21.1,<1.0
```

Comment:
`drf-spectacular>=0.21.1,<1.0` should move down after `django-cors-headers>=3.10.1,<4.0`

Reply:
resolved